### DISCUSSION TITLE
radio 选中后，中心点偏移

### DISCUSSION THREAD

**Discussion by wenmingwei** [ant-design/ant-design#43272] [Q&A] [answered]
antd版本：4.20.0

问题现象：
![1](https://github.com/ant-design/ant-design/assets/40015360/2c8db488-dfcf-4b7b-b286-90dd504aced7)

问题重现：
1. 首先进入页面A，页面A中包含radio，显示正常
2. 进入页面B，页面B通过iframe集成了一个使用了不同版本antd的react应用
3. 返回到页面A，页面A中的radio显示异常

原因分析：
中心点:after的position计算出现了偏差，但是css中position均为absolute

CSS及radio的截图：
![2](https://github.com/ant-design/ant-design/assets/40015360/9ad50787-2ec7-4d0e-b08d-cf6f6d4cfbcf)

![3](https://github.com/ant-design/ant-design/assets/40015360/8db04434-d84e-43dc-9ea6-848cf3f8598c)

**Comment by MadCcc** [comment, marked as answer]
iframe 不会造成样式污染。如果有样式污染，将应用中使用的 antd 修改 prefixCls 即可

**Reply by wenmingwei** [reply]
找到了原因，不是因为iframe导致的，是iframe所在的页面引入了一个开源的npm包；
里面使用了不同版本的antd-css，导致了该问题。

使用prefixCls确实可以解决问题。
